### ISSUE TITLE
update Rochester correction to v3 for 2017

### ISSUE THREAD

**Issue by bortigno** [UFLX2MuMu/Ntupliser#61]
New v3 Rochester corrections available for 2017
https://twiki.cern.ch/twiki/bin/viewauth/CMS/RochcorMuon

**Comment by bortigno** [comment]
I open a new branch issue-61-roch-cor-v3-2017.
It is just matter of updating the data file. I copied the RoccoR2017.txt to the RochCor/data/ folder. This must be update in the plugins where apparently the configuration is hardcoded. I should remove this from the plugins and put it in the config (will open a separate issue for 2017 and 2018).

I also changed the output file name, using the input sample.name.
Compiling and testing with the new corrections. test on data and MC run without crashing or strange messages.

I now copied plotMass.py from the 2018 folder and run it to validate the new Rochester Corrections on a Higgs sample. This is the result:

**Rochester Corrections v3  (in red)**
![Screenshot 2019-03-26 at 16 02 28](https://user-images.githubusercontent.com/8289819/55007961-91dd9f80-4fe0-11e9-9768-6be1d78e0a24.png)

**Rochester Corrections v2  (in red)**
![Screenshot 2019-03-26 at 16 03 42](https://user-images.githubusercontent.com/8289819/55008058-bc2f5d00-4fe0-11e9-8b22-c2915458bf43.png)

They look similar enough that I am confident to open a PR and merge it directly.
PR is #62
